### PR TITLE
feat: fixed quote and decimals in dashboard analytics

### PR DESCRIPTION
**Pull request by muradmalik777**

### CODE REVIEW

File: src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.js [hunk 1]
```diff
@@ -184,7 +184,10 @@ const ProfitSharingAnalytics = ({ provider }) => {
       ) : (
         balanceHistory && (
           <>
-            <TradingPerformance performance={stats.performance} unit=" USDT" />
+            <TradingPerformance
+              performance={stats.performance}
+              unit={balanceHistory && balanceHistory.quote ? balanceHistory.quote : ""}
```
Comment (cwagner22) [inline src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.js:189]: You can just to `balanceHistory?.quote`
or `balanceHistory?.quote || ''` if needs to be empty string by default

**Review by cwagner22 — APPROVED** [review]

File: src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.js (hunk 1 above)
Comment (muradmalik777) [reply src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.js:189]: well balanceHistory is null initially thats why i added this.

Comment (cwagner22) [reply src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.js:189]: Yeah but the syntax I gave you takes care of that: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Comment (muradmalik777) [reply src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.js:189]: it does? okay changing it

Comment (muradmalik777) [reply src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.js:189]: Done.

**Pull request merged**